parsers/go.c

```diff
@@ -968,6 +968,8 @@ static void parseConstTypeVar (tokenInfo *const token, goKind kind, const int sc
 	// VarSpec     = IdentifierList ( Type [ "=" ExpressionList ] | "=" ExpressionList ) .
 	bool usesParens = false;
 	int member_scope = scope;
+	intArray *corks
+		= (kind == GOTAG_VAR || kind == GOTAG_CONST)? intArrayNew (): NULL;
```

Comment:
I can avoid this array allocation with using file static variable (or function static variable).

Reply:
I decide to keep this allocation as is.